sections/06_discussion.md

```diff
@@ -6,9 +6,28 @@ can be further pruned.*
 
 ### Evaluation
 
-The human genome has 3E9 base pairs (bp) and only a small fraction of them are implicated in specific biochemical activities. As a result, binary classification of genomic regions based on their biochemical activity results in small positive class and a large negative class with important consequences for model evaluation.  For example, based on processed ChIP-seq data, the number of reproducible transcription factor (TF) binding sites genome-wide is typically in the 1,000-50,000 range. ChIP-seq data allows TF binding site detection with ~200 bp resolution and thus less than 1% of the genome can be confidently labeled as bound for most transcription factors. Therefore, it is important evaluate the genome-wide recall and false discovery rates (FDR) of computational models of TF binding. TF binding site predictions may be used for targeted validation experiments and as features for models of downstream processes such as gene expression. Targeted experimental validation usually necessitates an FDR in the 5%-25% range. For downstream modeling, a low FDR for TF binding predictions is not as critical if the downstream models meet their respective recall and and FDR requirements; an FDR of 50% in this context may suffice.
+What are the challenges in evaluating classification models that are specific to this domain? The human genome has 3E9 base pairs (bp) and only a small fraction of them are implicated in specific biochemical activities. As a result, classification of genomic regions based on their biochemical activity results in highly imbalanced classification. What are appropriate evaluation metrics that account for the label imbalance? The classification labels are formulated based on continuous value experimental signals - what is an appropriate procedure for formulating binary classification labels based on these signals? The experimental signals are only partially reproducible across experimental replicates - what is an appropriate upper bound for classification performance that accounts for the experimental reproducibility?
```

Comment:
This intro works much better.  It might fit the rest of the discussion style better to turn some of these questions into statements.

The domain we cover is very broad - from medical records to genomics to protein structure - so I suggest we start even broader before zooming in to TFs.

What do you think of replacing the first sentence with something like:
> There are unique challenges to evaluating deep learning predictions in the biomedical domain.  We focus on TF binding prediction as a representative task to illustrate some of these issues.

We don't use "base pairs" again so you can drop the "(bp)" abbreviation.

After "highly imbalanced classification" please add "...classification, which also arises in other problems we review such as virtual screening for drug discovery".